app.py

```diff
@@ -18,32 +18,24 @@ def index():
 
 @app.route('/get_tweets.json', methods=['GET'])
 def get_tweets():
-    # import cron_pull_tweets
-
-    if os.path.isfile('secrets.txt'):
-        with open('secrets.txt', 'r') as f:
-            params = json.load(f)
-    else:
+    if 'PRODUCTION' in os.environ:
```

Comment:
What about a key-value variable like `'APP_ENV': 'PRODUCTION'` instead?

Reply:
Will do! Should I cancel this pull request and refactor, or fix it in the next commit?